DictionaryTypes.txt

```diff
@@ -266,6 +266,7 @@ enum(ch_type) "Type of the channel"
     hpi_error	    708		     "HPI fit error"
     hpi_movement    709		     "HPI coil movement"
     seeg            802              "Stereo EEG channel"
+    dbs             803              "Deep brain stimulation channel"
```

Comment:
@jnenonen @mkajola okay with you? The reasoning is that the iEEG-BIDS spec has three channel types: ECoG, sEEG, and DBS. We already have the other two ECoG and sEEG, so this just completes the spec. I'm not sure why we made ECoG 902, but sEEG is 802 and is pretty similar to DBS so 803 was chosen here.

For reference, iEEG-BIDS pub: https://www.nature.com/articles/s41597-019-0105-7

Reply:
Sounds fine. Done.